src/libawkward/array/ListOffsetArray.cpp

```diff
@@ -270,7 +270,7 @@ namespace awkward {
 
     int64_t carrylen = offsets.getitem_at_nowrap(offsets.length() - 1);
     Index64 nextcarry(carrylen);
-    struct Error err = util::awkward_listarray_broadcast_tooffsets64<T>(
+    struct Error err = kernel::listarray_broadcast_tooffsets64<T>(
```

Comment:
Similar to above, both the kernel dispatch and underlying kernel need an underscore in their names: `..._broadcast_tooffsets_64`.